### DISCUSSION TITLE
pstt hey you

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#131] [Show and tell]
FELLOW TOONSQUID USERS !! HI !! how are you all doing this fine day ? yall should show off ur works in the show and tell category :D I'd love to see some of yalls work made in toonsquid !! heres a wip of mine :D 

https://user-images.githubusercontent.com/115050959/212765160-81918da3-c0ea-4c3a-94b3-bdaf6cc4f4ad.mov

**Comment by DinoAwakening** [comment]
Probably not gonna do that. Almost all I make is NSFW stuff tho… xd

**Reply by trinhpxl** [reply]
oh damn thats tragic xDD

**Reply by DinoAwakening** [reply]
Still love your animation, you did such a great work B)

**Reply by trinhpxl** [reply]
ah thank u !! :]
